### ISSUE TITLE
Incorrect unit  analysis on `quotes` property

### ISSUE THREAD

**Issue by bartveneman** [projectwallace/css-analyzer#325]
```css
q{quotes:2 1C 2 1D 2 18 2 19}
```

Source: syntax.fm

`C` and `D` are reported as units, which they aren't.

**Comment by bartveneman** [comment]
I guess this is what you get for writing incorrect CSS. Using the analyzer to catch these sort of issues is more of a feature than a bug, therefore closing.
